### ISSUE TITLE
Table verb errors

### ISSUE THREAD

**Issue by rickdonato** [netenglabs/suzieq#634]
**Describe the bug**
`table show` returns 404

**To Reproduce**
```
rick> table show
  errpr
0   404
rick> table summarize
  errpr
0   404
```
**Attach**
TBC
 **Expected behavior**
TBC
**Screenshots**
TBC
**Additional context**
Environment from which this is occurring in:
* remote cli is being used
* correct port setting applied via "set rest-server-port=443"
* can successfully query other tables. For example "device show"

When using Local DB the error changes to:
```
root> table show
                                    error
0  ERROR: Table sqpoller-output not found
```

**Comment by rickdonato** [comment]
Sure for the client or server?

**Comment by ddutt** [comment]
For the server

**Comment by ddutt** [comment]
Fixed in 0.17.1 via commit 06507652eac6888ea808cd574fb0c0f9906fc42e
